ts-redis/index.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @klotho::execution_unit {
+ *  id = "redis-main"
+ * }
+ */
+
+import * as express from "express";
+import { createClient } from 'redis';
+
+/**
+* @klotho::persist {
+*   id = "redis"
+* }
+*/
+const client = createClient({});
+
+const app = express();
+app.use(express.json());
+
+app.post("/user/", async (req, res) => {
+  const key = req.body["firstName"];
+  const value = req.body["lastName"];
+  await client.connect();
```

Comment:
Should this be connecting on every request? This pattern I generally don't like because it's easy to forget - if the `connect` can be long-lived, then I'd prefer the `setupClient` type approach that returns a `Promise<redis.Client>` or something. See the `ts-sequelize` sample for that pattern.

Reply:
i can, didnt realize it was an option, Will change now